### DISCUSSION TITLE
Praise for phenomenal library

### DISCUSSION THREAD

**Discussion by pikeas** [rgossiaux/svelte-headlessui#123] [General]
Thanks for making exactly what I was looking for! And you went even further - your documentation is some of the best and most thorough I've seen in any frontend project.

Keep up the good work!

**Comment by rgossiaux** [comment]
Thanks for the kind words! I can't take full credit for the documentation, though--it was adapted with permission from https://headlessui.dev, though I did still end up editing and rewriting quite a bit to adapt it to Svelte.
